frontend/src/component/application/ApplicationOverview.tsx

```diff
@@ -46,37 +85,92 @@ export const ApplicationOverview = () => {
         ],
     };
 
-    const applicationNode = `
-    ${applicationName}[<span>${applicationName}</span>]
-    `;
-
     // @ts-ignore
     window.navigateToInstances = (environment: string) => {
```

Comment:
This can also be removed now

Reply:
Btw, the bottom nodes should all navigate to relevant environment eventually.

Reply:
Yup Nico told me about this. We'll make those links in the next PR